.github/workflows/build.yml

```
@@ -0,0 +1,15 @@
+name: CI
+
+on: [push, pull_request]
```

Don't need both push and pull_request
```suggestion
on: [push]
```